socialreview/server/component-config.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Created this file to be able to use the StrongLoop explorer at http://localhost:3000/explorer